### ISSUE TITLE
Large functions cause `constants error

### ISSUE THREAD

**Issue by cruzzil** [LeslieGoldsmith/qprof#1]
Large function definitions cause a `constants exception when running .prof.prof`

**Comment by LeslieGoldsmith** [comment]
Yes, I can see how that could happen (as well as a few other limit-related parsing errors, such as globals or branch  distance). How many lines does your function have? And how close to the limit of 95 constants was it?
